systemverilog-plugin/UhdmAst.cc

```diff
@@ -2173,13 +2174,64 @@ void UhdmAst::process_typespec_member()
     add_multirange_wire(current_node, packed_ranges, unpacked_ranges);
 }
 
-void UhdmAst::process_enum_typespec()
+void UhdmAst::process_enum_typespec(const UHDM::BaseClass *object)
```

Comment:
I think you should be able to get ``UHDM::BaseClass`` from ``obj_h`` that is already available in this function using:
```
    const uhdm_handle *const handle = (const uhdm_handle *)obj_h;
    const UHDM::BaseClass *const object = (const UHDM::BaseClass *)handle->object;
```

Have you tried this?

Reply:
At first I didn't realize it could be better not to pass a parameter to the function, so I didn't use `obj_h`, but now I've modified this like you suggested and it works indeed 👍🏻.